packages/design-system-docs/src/pages/components/Review/Review.example.tsx

```diff
@@ -1,8 +1,8 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import { Review } from '@design-system';
+import { Review } from '@design-system/packages/design-system';
 
-const editButtonHref = 'javascript:void(0);';
+const editButtonHref = '#!';
```

Comment:
Fixes JS error, react deprecating use of scripts in hrefs

Reply:
What does the `!` do?

Reply:
It references the id = ! which doesn't exist so it's similar to # but doesn't cause anything to happen, where using # on it's own still throws an error in the console based on the react dep warning.. so it's a bit of a hack to make it work without an error.. best way I know of!

Reply:
Oh that's interesting! TIL